0-ad-history-mod-package/simulation/templates/units/sele/siege_ram.xml

```diff
@@ -7,6 +7,12 @@
     <Civ>sele</Civ>
     <SpecificName>Krios</SpecificName>
     <Icon>units/mace/siege_ram.png</Icon>
+    <History>
+        The krios (battering ram) was one of the most popular siege weapons in ancient Greece, and was used to damage walls or make them collapse, which was basically impossible without it.
+        A 65- to 150-foot-long pole was hung by ropes on a horizontal beam. It was pulled back, released, and then swung forward, hitting the wall at maximum speed. The people operating the ram were protected by a roofed shell. The whole construction rolled on wheels because it was too heavy to be carried.
```

Comment:
```suggestion
        A 65- to 150-foot-long pole was hung by ropes on a horizontal beam. It was pulled back and released. The pole swung forward, hitting the wall at maximum speed. The people operating the ram were protected by a roofed shell. The whole construction rolled on wheels because it was too heavy to be carried.
```